src/graph/session/GraphSessionManager.cpp

```diff
@@ -276,14 +304,45 @@ Status GraphSessionManager::init() {
       continue;
     }
     session.queries_ref()->clear();
+    std::string key = session.get_user_name() + session.get_client_ip();
     auto sessionPtr = ClientSession::create(std::move(session), metaClient_);
     auto ret = activeSessions_.emplace(sessionId, sessionPtr);
     if (!ret.second) {
       return Status::Error("Insert session to local cache failed.");
     }
+    bool addResp = addSessionCount(key);
+    if (!addResp) {
+      return Status::Error("Insert userIpSessionCount to local cache failed.");
+    }
     updateSessionInfo(sessionPtr.get());
+    loadSessionCount++;
   }
+  LOG(INFO) << "Total of " << loadSessionCount << " sessions are loaded";
   return Status::OK();
 }
+
+bool GraphSessionManager::addSessionCount(std::string key) {
+  auto countFindPtr = userIpSessionCount_.find(key);
+  if (countFindPtr != userIpSessionCount_.end()) {
+    countFindPtr->second.get()->fetch_add(1);
+  } else {
+    auto ret1 = userIpSessionCount_.emplace(key, std::make_shared<SessionCount>());
+    if (!ret1.second) {
+      return false;
+    }
+  }
+  return true;
+}
+
+void GraphSessionManager::subSessionCount(std::string key) {
```

Comment:
`void GraphSessionManager::subSessionCount(const std::string& key)`

Maybe consider changing the return type to bool to indicate the case that the key is not found?